ext/pcntl/tests/pcntl_signal_001.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+pcntl_signal() signal greater than max available.
+--EXTENSIONS--
+pcntl
+--FILE--
+<?php
+pcntl_signal(1000000, function($signo){
+    echo "signaled\n";
+});
+echo "ok\n";
```

Comment:
Please use a try/catch and show the error message, this would allow it to use an EXPECT section instead of EXPECTF with ``%A``

Reply:
Ok I Will adjust this. Please wait for My change

Reply:
(I need to format my PC today and I get back to this, sorry for the delay)